src/main/java/com/linkedin/kmf/partitioner/Partitioner.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.partitioner;
+
+public interface Partitioner {
```

Comment:
Can you name the interface `KMPartitioner` to avoid class name conflicts with open source classes, similar to `KMBaseProducer`?